src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/client/PrisonApiClient.kt

```diff
@@ -46,24 +42,6 @@ class PrisonApiClient(
       }
       .block(apiTimeout)
   }
-
-  fun getPrisonerDetails(offenderNo: String): PrisonerDetailsDto? {
```

Comment:
this call is no longer needed as we were doing a call to both Prison API and PrisonerOffenderSearch. the same data also comes back in the call to PrisonerOffenderSearch so will use the same.